<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchMIDI</class>
 <widget class="QWidget" name="TouchMIDI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>201</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TouchMIDI</string>
  </property>
  <widget class="QWidget" name="CTLWheelWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="CTL1_Type">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>50</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Springs">
     <item>
      <widget class="QCheckBox" name="Pitch_Spring">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="MOD_Spring">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CTL1_Spring">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CTL2_Spring">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>172</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Spring</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>30</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="Wheels">
     <item>
      <widget class="QSlider" name="Pitch">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <number>-8191</number>
       </property>
       <property name="maximum">
        <number>8191</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="MOD">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="CTL1">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="CTL2">
       <property name="maximum">
        <number>127</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>30</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CTL 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>90</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CTL 2</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>PITCH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MOD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CTL 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CTL 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="CTL2_Type">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>110</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="RelativeCheck">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Relative Mode</string>
    </property>
    <property name="text">
     <string>Rel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="XYWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>250</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="CTLX_Type">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>26</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>6</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CTL X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>56</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CTL Y</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CTLY_Type">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>76</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>6</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="CTLZ_Type">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>126</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>106</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CTL Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="XVal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>6</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>56</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>106</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="YVal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>56</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ZVal">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>106</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="XType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>26</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Note</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Velocity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ctl</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="YType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>76</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Note</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Velocity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ctl</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="ZType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>126</y>
      <width>89</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Note</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Velocity</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ctl</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="note_high">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>162</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>84</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="note_low">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>162</y>
      <width>55</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>127</number>
    </property>
    <property name="value">
     <number>48</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>36</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Note Range</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>172</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Range</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="Mode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Control Wheel</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XY Table</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="AboutButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
